float64[] averaged_scan
